src/ui/A/A.jsx

```diff
@@ -13,6 +13,7 @@ const baseClass = `
 `
 const variantClasses = {
   default: `text-ds-blue`,
+  newHeader: `font-semibold`, // clean up when old header gets removed
```

Comment:
nit: Not needed for this time, but maybe can call header deprecated_header and this as "header" next time around

Reply:
yea I considered that - so much work though 🥵 

To punish myself for this laziness I will fix it now